charts/operator-wandb/charts/bufstream/templates/deployment.yaml

```diff
@@ -36,8 +36,7 @@ spec:
           {{- toYaml . | nindent 8 }}
         {{- end }}
     spec:
-      imagePullSecrets:
-      {{- include "wandb.imagePullSecrets" . }}
+      {{- include "wandb.imagePullSecrets" . | nindent 6 }}
```

Comment:
I think this is different than the standard way helm specifies this.  I think the way helm charts do this by default is using
```
{{- with .Values.imagePullSecrets }}
imagePullSecrets:
  {{- toYaml . | nindent 8 }}
{{- end }}
```
This removes the need for the helper template but expects the `imagePullSecrets` to be an array of maps.